src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -720,12 +722,40 @@ def calibrate(self, *inputs: numpy.ndarray) -> numpy.ndarray:
         if isinstance(raw_result, RawOpOutput):
             return raw_result
 
-        quantized_samples = QuantizedArray(self.n_bits, raw_result)
+        supports_analytical_quant = all(
```

Comment:
In theory all mixing ops could be calibrated analytically. But only Gemm implements the analytical formulas, all others will throw some kind of E_NOT_IMPL